server/helpers/utilities.js

```diff
@@ -110,7 +110,7 @@ const hmppsAuthMFAUser = (token) => jwtDecode(token).authorities.includes('ROLE_
 
 const getSnoozeUntil = (rawSnoozeUntil) => {
   const snoozeUntil = moment(rawSnoozeUntil)
-  return snoozeUntil.isAfter(moment()) ? snoozeUntil.format('dddd, Do MMMM, YYYY') : ''
+  return snoozeUntil.isAfter(moment()) ? snoozeUntil.add(1, 'day').format('dddd, Do MMMM YYYY') : ''
```

Comment:
Do you want `moment().endOf('day')` here?

Reply:
It either needs to say 'Until 23:59:59 Monday' or 'Until Tuesday' IMO

Reply:
Not in this case. This represents a date to be displayed when the notifications will resume. 
`EndOf` would return the end of the day albeit by a second, but the format applied would still display the previous day and not the next one. Or at least thats my understanding.

Reply:
I wrote that then panicked and went off to check; https://momentjs.com/docs/#/manipulating/end-of/
My reading of that, is it still returns the previous day 🤔